.github/workflows/ci.yml

```
@@ -9,21 +9,25 @@ on:
     tags: '*'
 jobs:
   test:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    name: Julia ${{ matrix.version }}/${{ matrix.os }}/${{ matrix.arch }}/group ${{ matrix.group }}/${{ matrix.threads }} threads/${{ github.event_name }}
```

In the details page, what's visible is "Julia 1.4/ubuntu-latest/x64/gro...". Maybe this will work to put the details which vary first?
```suggestion
    name: v${{ matrix.version }}/t-${{ matrix.threads }}/group-${{ matrix.group }}/${{ github.event_name }}/${{ matrix.arch }}+${{ matrix.os }}
```